### ISSUE TITLE
unclear error message in PythonCallable

### ISSUE THREAD

**Issue by edublancas** [ploomber/ploomber#961]
post-mortem debugging is not supported in PythonCallable (only in NotebookRunner). However, the docs and the error message do not reflect that. We should ensure that this is clear in the docs and that when passing `debug='pm'` to a PythonCallable, the error says:

```
'pm' is invalid. Post-mortem debugging is only supported in notebook tasks
```

**Comment by edublancas** [comment]
see https://github.com/ploomber/ploomber/issues/958

**Comment by edublancas** [comment]
I fixed this this as part of the new debugging feature that will be released in 0.20
